srcpkgs/android-studio/template

```diff
@@ -1,40 +1,60 @@
 # Template file for 'android-studio'
 pkgname=android-studio
-version=3.4.2
+version=3.5.2
 revision=1
 # _studio_build and _studio_rev are for downloading the zip from dl.google.com
 # https://developer.android.com/studio/#resources as of 2018-07-12
-_studio_build=183.5692245
+_studio_build=191.5977832
 _studio_rev=0
 archs="x86_64 i686"
-create_wrksrc=yes
 depends="gtk+ libGL"
 short_desc="Official Android IDE"
 maintainer="Jordyn Carattini <onlinecloud1@gmail.com>"
 license="Apache-2.0"
 homepage="http://tools.android.com/"
 # changelog="https://developer.android.com/studio/releases/index.html"
 distfiles="https://dl.google.com/dl/android/studio/ide-zips/${version}.${_studio_rev}/android-studio-ide-${_studio_build}-linux.tar.gz"
-checksum=35eb8c74837d1aab59229101fc91568a607ac04854a40209f7a0ba7ac0601924
+checksum=f838486ce847db802bdaf1163059033934146c6ccdcdaa9a398bd85cda348d4d
 repository=nonfree
+restricted=yes
```

Comment:
Just curious, why is Android-studio now restricted?

Reply:
I believe the binary distribution should've been restricted from the start. At least now, the EULA provided by Android Studio says this:

> 3.4 You may not use the SDK for any purpose not expressly permitted by the License Agreement.  Except to the extent required by applicable third party licenses, _you may not copy (except for backup purposes), modify, adapt, redistribute, decompile, reverse engineer, disassemble, or create derivative works of the SDK or any part of the SDK._

Reply:
Ah ok, I see.